multitenancy/tenant-manager-api/src/main/java/io/apicurio/multitenant/storage/RegistryTenantStorageImpl.java

```diff
@@ -42,16 +45,29 @@ public void save(RegistryTenantDto dto) {
         return repo.find("tenantId", tenantId).singleResultOptional();
     }
 
-    @Override
-    public List<RegistryTenantDto> listAll() {
-        return repo.listAll();
-    }
-
     @Override
     public void delete(String tenantId) {
         RegistryTenantDto dto = findByTenantId(tenantId)
             .orElseThrow(() -> TenantNotFoundException.create(tenantId));
         repo.delete(dto);
     }
 
+    @Override
+    public List<RegistryTenantDto> queryTenants(String query, Sort sort, Parameters parameters,
+            Integer offset, Integer returnLimit) {
```

Comment:
Check for negative values of `offset` and `limit`.

Reply:
it's already checked in the rest layer